website/docs/en/blog/announcing-0.6.mdx

```diff
@@ -0,0 +1,198 @@
+---
+date: 2024-04-10 16:00:00
+---
+
+import { PackageManagerTabs } from '@theme';
+
+# Announcing Rspack v0.6
+
+> April 10, 2024
+
+## Major Feature Updates
+
+### Built-in support for mini-css-extract-plugin
+
+Now you can use `rspack.CssExtractRspackPlugin` as a replacement for `mini-css-extract-plugin`.
+
+This is very useful in some scenarios, for example when the built-in CSS parser cannot meet your needs, there are more customized CSS module names, or you want to use some loaders that depend on the output of css-loader, but still want to extract the CSS into a separate file.
+
+For more details, please see [CssExtractRspackPlugin](plugins/rspack/css-extract-rspack-plugin.html#cssextractrspackplugin)
```

Comment:
@JSerFeng can you help add a css-extract-plugin example in rspack-example and link it here

Reply:
https://github.com/rspack-contrib/rspack-examples/pull/46